Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -835,21 +836,22 @@ namespace Babylon
                 , m_jsTransform{Napi::Persistent(XRRigidTransform::New(info))}
                 , m_jsViews{Napi::Persistent(Napi::Array::New(info.Env(), 0))}
                 , m_transform{*XRRigidTransform::Unwrap(m_jsTransform.Value())}
+                , m_isEmulatedPosition{true}
             {
             }
 
-            void Update(const Napi::CallbackInfo& info, gsl::span<const xr::System::Session::Frame::View> views)
+            void Update(const Napi::CallbackInfo& info, const xr::System::Session::Frame* frame)
```

Comment:
I strongly recommend against taking a raw pointer as an argument unless it is intended that this argument can genuinely be null. Since I don't believe this function should ever be called on `nullptr`, I recommend changing this signature to `const xr::System::Session::Frame&` and dereferencing in the calling code.

Reply:
You are correct that in the context the frame is used here it should never be null. I've updated it to pass it by reference now rather than pointer.